tools/rapids-wheel-ctk-name-gen

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# A utility script that generates CUDA suffix
```

Comment:
Would be good to add small documentation of what the CUDA suffix looks like so that it's clear that we do `cu11` rather than `cuda11`/`cuda_11`/etc.